crates/fuel-core/src/combined_database.rs

```diff
@@ -215,7 +219,7 @@ impl CombinedDatabase {
         )?;
         let relayer = Database::open_rocksdb(
             path,
-            StateRewindPolicy::NoRewind,
+            state_rewind_policy,
```

Comment:
is this change intended? curious

Reply:
Yes it is. Having a hard-coded `NoRewind` here kind of defeats the purpose of supporting state rewinds. Though an improvement would be to support a separate argument for the relayer state rewind policy since DA blocks are produced less frequently, but I don't want to prematurely bloat the method interface for that since that could be a subsequent optimization.